src/openstudio_lib/LocationTabView.hpp

```diff
@@ -31,6 +31,32 @@ class Site;
 class YearDescription;
 }  // namespace model
 
+/// <summary>
+/// SortableDesignDay is a helper class for sorting DesignDays
+/// </summary>
+class SortableDesignDay
+{
+ public:
+  explicit SortableDesignDay(const openstudio::model::DesignDay& designDay);
+
+  static int qstringToPermil(const QString& str);
+
+  static QString permilToQString(int permil);
+
+  static QString key(const QString& type, int sortablePermil);
+
+  QString type() const;
```

Comment:
@jmarrec it might be useful to add this interface directly to the SDK DesignDay object, basically it would just parse this information out of the DesignDay name.  What do you think?  Is that useful enough I should make a PR to the SDK?

Reply:
I missed this comment, sorry. Yes, that'd be helpful I think.